examples/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set(ALL_EXAMPLES sum_views)
+
+message("Examples to be built: ${ALL_EXAMPLES}")
+
+foreach(example ${ALL_EXAMPLES})
+    add_executable(beman.any_view.examples.${example})
```

Comment:
this seems way to verbose -- the executable should just be the example name -- it's already going into a subdirectory

Reply:
solved!